src/main/java/ch/njol/skript/SkriptEventHandler.java

```diff
@@ -335,9 +335,9 @@ public static void unregisterBukkitEvents(Trigger trigger) {
 
 	/**
 	 * Events which are listened even if they are cancelled. This should no longer be used.
-	 * @deprecated Users should specify the listening behavior in the event declaration. "on any %event%:", "on cancelled %event%:".
+	 * @deprecated Users should specify the listening behavior in the event declaration. "on any %event%:", "on cancelled %event%:". 
 	 */
-	@Deprecated
+	@Deprecated(since = "2.11.0", forRemoval = true)
```

Comment:
This was like 2.7
You can check by looking at the git blame instead of assuming 2.11